tools/editor/editor_data.cpp

```diff
@@ -260,6 +260,19 @@ EditorPlugin* EditorData::get_subeditor(Object *p_object) {
 	return NULL;
 }
 
+Vector<EditorPlugin*> EditorData::get_subeditors(Object* p_object) {
+    
+    Vector<EditorPlugin*> sub_plugins;
```

Comment:
This chunk still uses spaces as indentation (also make sure to remove trailing spaces on empty lines like l. 264 and 266).